app/views/guests/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Guest Name: <%= @guest.name %></h1>
+
+<%= render "layouts/rooms_info" %>
```

Comment:
Nice touch, however I think you're only using this once, so technically no need for the abstraction yet, but nice to see you know how it works!